core/app/models/spree/promotion.rb

```diff
@@ -166,22 +168,20 @@ def products
     # Whether the promotion has exceeded it's usage restrictions.
     #
     # @return true or false
-    def usage_limit_exceeded?
+    def usage_limit_exceeded?(excluded_orders = [])
```

Comment:
We need to also update the YARD documentation here.

Reply:
Also, can we use a keyword argument here so it's clear, even when this method is called, that the array contains orders to exclude? I feel like this 

```
usage_limit_exceeded?([promotable])
```

could be better for the reader in this form:

```
usage_limit_exceeded?(excluded_orders: [promotable])
```

Reply:
> We need to also update the YARD documentation here.

Missed that 😱 



> can we use a keyword argument here so it's clear, even when this method is called, that the array contains orders to exclude?

Absolutely agree, I used a positional argument just to be consistent with `#used_by?` but here I'd prefer a keyword one too.